### ISSUE TITLE
HTTP

### ISSUE THREAD

**Issue by cup** [marcusz/floyd#73]
it seems language currently has no way to make an HTTP request or act as an HTTP client

- http://rosettacode.org/wiki/HTTP
- https://programming-idioms.org/idiom/101

**Comment by marcusz** [comment]
True! It's an important feature and it's on the todo list but no one is working on it right now. I currently work on the LLVM backend.
